src/main/java/geekbrains/java2/ht1b/Main.java

```diff
@@ -113,13 +122,11 @@ private static boolean validMove(String move){
             return true;
         if(move.equalsIgnoreCase("C2") && (board[2][1] == "C2"))
             return true;
-        if(move.equalsIgnoreCase("C3") && (board[2][2] == "C3"))
-            return true;
-        return false;
+        return move.equalsIgnoreCase("C3") && (board[2][2] == "C3");
```

Comment:
\[Checkstyle\] ERROR: Literal Strings should be compared using equals\(\), not '=='\.